app/client/components/productSwitch/CancellationSwitchConfirmed.tsx

```diff
@@ -0,0 +1,274 @@
+import { css } from '@emotion/react';
+import {
+	space,
+	neutral,
+	headline,
+	textSans,
+	brand,
+	brandAlt,
+} from '@guardian/source-foundations';
+import {
+	Button,
+	Stack,
+	SvgArrowRightStraight,
+} from '@guardian/source-react-components';
+import { useNavigate, Link, useLocation, Navigate } from 'react-router-dom';
+import { minWidth } from '../../styles/breakpoints';
+import GridPicture from '../images/GridPicture';
+import GridImage from '../images/GridImage';
+import { useContext } from 'react';
+import {
+	ProductSwitchContext,
+	ProductSwitchContextInterface,
+	ProductSwitchResponse,
+} from './productSwitchApi';
+import { getGeoLocation } from '../../geolocation';
+import {
+	getIosAppUrl,
+	productStartDate,
+	productFirstPaymentAmount,
+} from './productSwitchHelpers';
+import { CancellationRouterState } from '../cancel/CancellationContainer';
+import { headingCss, listCss, standfirstCss } from './productSwitchStyles';
+
+const CancellationSwitchConfirmed = () => {
+	const navigate = useNavigate();
+
+	const productSwitchContext = useContext(
+		ProductSwitchContext,
+	) as ProductSwitchContextInterface;
+	const location = useLocation();
+	const routerState = location.state as CancellationRouterState;
+	const productSwitchConfirmationInfo =
+		routerState?.productSwitchConfirmationInfo as ProductSwitchResponse;
+
+	if (!productSwitchConfirmationInfo) {
+		return <Navigate to="/" />;
+	}
+
+	const newProduct = productSwitchConfirmationInfo.newProduct;
+
+	return (
+		<>
+			<h2 css={headingCss}>Your {newProduct.name} is now active</h2>
+			<Stack space={9}>
+				<p css={standfirstCss}>
+					Your {newProduct.billing.frequency.name}ly{' '}
+					{productSwitchContext.productType.friendlyName} has
+					successfully been changed to a {newProduct.name}. We’ve
+					stopped your previous payments and started you on your new
+					plan. Please check your inbox for an email containing all
+					your details and information on how to access your benefits.
+				</p>
+				<div
+					css={css`
+						border: 1px solid ${neutral[86]};
+					`}
+				>
+					<div
+						css={css`
+							background-color: #e3edfe;
```

Comment:
could this be `palette.brand[800]`

https://guardian.github.io/source/?path=/story/packages-source-foundations-palette--page